packages/custom-elements/tests/html/Element/setAttribute.html

```diff
@@ -43,6 +46,7 @@
 
 suite('Setting new values.', function() {
   let localName1;
+  let localName2;
```

Comment:
Was this just left out before?

Reply:
Yeah, converting the test suite to a module / strict mode exposed this.